DJHelper/DJHelper/Model/Event Model/Event Controller/EventController.swift

```diff
@@ -283,7 +288,7 @@ class EventController {
         encoder.dateEncodingStrategy = .iso8601
 
         do {
-            try CoreDataStack.shared.save()
+//            try CoreDataStack.shared.save()  // commented out this line because it's redundant with later in the method
```

Comment:
I agree, I think you put this in so if you want you can just delete it all together.